esmvalcore/preprocessor/_regrid.py

```diff
@@ -1162,7 +1152,7 @@ def _rechunk_aux_factory_dependencies(
     cube_chunks = cube.lazy_data().chunks
     for coord in factory.dependencies.values():
         coord_dims = cube.coord_dims(coord)
-        if coord_dims is not None:
+        if coord_dims:
```

Comment:
I did want to mention that this was the actual change I made; all the other changes were introduced by the pre-commit hooks 😊